01_02_Combat_Rules.txt

```diff
@@ -186,6 +186,26 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
 	A target reduced to exactly 1 health by a nonlethal attack does not 
 fall Unconscious. 1 is not 0.
 
+Melee Accuracy and Guard Critical Hits:
+	See "Critical Hits" below for general Critical information. Melee Critical 
+hits deal extra damage as normal. Critical Guard rolls block all damage from any 
+non-Extra-critical melee attack. Extra-critical Guard rolls block all damage from 
+the incoming attack. 
+
+	Melee Accuracy rolls may Critically fail in the same way as firearms. 
+Unless otherwise specified, melee weapons have a Jam range of 2. Should your 
+melee weapon "jam", it catches on something in the environment at the DM's 
+discretion, if applicable, and is stuck. Unsticking the weapon requires a 
+Weapons - Melee check as a firearm would, except that if you pass the DC by 
```

Comment:
I'd go with "just as clearing a firearm jam requires a Weapons - [X] check"; it's possible to read the original as "unjamming a firearm takes a weapons - melee check"
Also, I'd split the "except that" clause into its own sentence.